go.work

```diff
@@ -0,0 +1,3 @@
+go 1.21.4
```

Comment:
is this supposed to be committed?

Reply:
apparently this shouldn't be committed: https://go.googlesource.com/proposal/+/master/design/45713-workspace.md